aspnetcore/tutorials/publish-to-azure-api-management-using-vs.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Publish an ASP.NET Core web API to Azure API Management with Visual Studio
+author: codemillmatt
+description: Learn how to publish an ASP.NET Core web API to Azure API Management using Visual Studio.
+ms.author: masoucou
+ms.custom: "devx-track-csharp, mvc"
+ms.date: 08/25/2020
+uid: tutorials/publish-to-azure-api-management-using-vs
+---
+# Publish an ASP.NET Core web API to Azure API Management with Visual Studio
+
+By [Matt Soucoup](https://twitter.com/codemillmatt)
+
+## Set up
+
+- Open a [free Azure account](https://azure.microsoft.com/free/dotnet/) if you don't have one.
+- [Create a new Azure API Management instance](/azure/api-management/get-started-create-service-instance) if you haven't already.
+- [Create a web API app project](first-web-api#create-a-web-project)
+
+## Configure the app
+
+Adding Swagger definitions to the ASP.NET Core web API allows Azure API Management to read the app's API definitions. Complete the following steps.
+
+### Add Swagger
+
+1. Add the [Swashbuckle.AspNetCore](https://www.nuget.org/packages/Swashbuckle.AspNetCore) NuGet package to the ASP.NET Core web API project:
+
+![Screenshot to configure the NuGet dialog](publish-to-azure-api-management-using-vs/_static/configure_nuget.png)
```

Comment:
Here's an example of how to fix the ordered list numbering.
```suggestion
	![Screenshot to configure the NuGet dialog](publish-to-azure-api-management-using-vs/_static/configure_nuget.png)
```

Reply:
Thanks Scott - those should be fixed now.